cvmfs/swissknife_sync.cc

```diff
@@ -582,6 +588,19 @@ int swissknife::CommandSync::Main(const swissknife::ArgumentList &args) {
   if (!this->InitDownloadManager(follow_redirects)) {
     return 3;
   }
+
+  if (!this->InitSignatureManager(params.public_keys, params.trusted_certs)) {
+    return 3;
+  }
+
+  UniquePtr<manifest::Manifest> manifest;
+  manifest = this->FetchRemoteManifest(params.stratum0,
+                                       params.repo_name,
+                                       params.base_hash);
+  if (!manifest) {
+    return 3;
```

Comment:
I expect that this can happen and would be the first point where a network glitch shows (e.g. with the S3 backend).  There should be an error message.